Core/Core/Features/Inbox/InboxSettings/View/InboxSettingsView.swift

```diff
@@ -0,0 +1,125 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2025-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct InboxSettingsView: View {
+    private let defaultPadding: CGFloat = 10
+    @ObservedObject private var viewModel: InboxSettingsViewModel
+    @Environment(\.viewController) private var controller
+    @FocusState private var focusedInput: FocusedInput?
+    private enum FocusedInput {
+        case signature
+    }
+
+    public init(viewModel: InboxSettingsViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        InstUI.BaseScreen(
+            state: viewModel.state
+        ) { geometry in
+            contentView(geometry: geometry)
+        }
+        .navigationBarTitleView(String(localized: "Inbox Signature", bundle: .core))
+        .navigationBarItems(trailing: doneButton)
+        .alert(String(localized: "Failed to load Inbox Settings", bundle: .core), isPresented: $viewModel.showFailedToLoadDialog) {
+            Button("Go back", role: .cancel) { viewModel.didTapBack.accept(controller) }
+            Button("Retry") { viewModel.didTapRefresh.accept(controller) }
+        }
+        .alert(String(localized: "Failed to save Inbox Settings", bundle: .core), isPresented: $viewModel.showFailedToSaveDialog) {
+            Button("Ok", role: .cancel) { }
```

Comment:
```suggestion
            Button("OK", role: .cancel) { }
```
Because we use it like this everywhere else